mingw-w64-zydis/PKGBUILD

```
@@ -0,0 +1,60 @@
+# Maintainer: Maksim Bondarenkov <maksapple2306@gmail.com>
+
+_realname=zydis
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=4.1.0
+pkgrel=1
+pkgdesc="Fast and lightweight x86/x86-64 disassembler and code generation library (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url="https://github.com/zyantific/zydis"
+license=('spdx:MIT')
+depends=("${MINGW_PACKAGE_PREFIX}-zycore-c")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
+             "${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-doxygen"
+             "${MINGW_PACKAGE_PREFIX}-python"
+             'git')
+source=("${url}/archive/v${pkgver}/${_realname}-${pkgver}.tar.gz"
+        "https://raw.githubusercontent.com/netadr/zydis/44eefca6ad7b124f0fce234483dcb555ce7b2a0e/resources/VersionInfo.rc")
+sha256sums=('31f23de8abb4cc2efa0fd0e827bbabcaa0f3d00fcaed8598e05295ba7b3806ad'
+            '4e4822ed30a7ec62e87d9628cc8215c7000f898e86f8e51cce6eb57fc5159404')
+
+prepare() {
+  cd "${_realname}-${pkgver}"
+
+  cp -f "${srcdir}/VersionInfo.rc" resources/VersionInfo.rc
```

```suggestion
  dos2unix "${_realname}/resources/VersionInfo.rc"
```

No need to copy it from another commit.